images/linux/ubuntu1804.json

```diff
@@ -54,10 +54,28 @@
         }
     ],
     "provisioners": [
+        {
+            "type": "shell",
+            "inline": [
+                "chmod -R 777 /usr/local/sbin"
+            ],
+            "execute_command": "sudo sh -c '{{ .Vars }} {{ .Path }}'"
+        },
+        {
+            "type": "file",
+            "source": "{{template_dir}}/scripts/base/apt-get-retry.sh",
+            "destination": "/usr/local/sbin/apt-get"
+        },
+        {
+            "type": "file",
+            "source": "{{template_dir}}/scripts/base/apt-retry.sh",
+            "destination": "/usr/local/sbin/apt"
+        },
         {
             "type": "shell",
             "inline": [
                 "mkdir {{user `image_folder`}}",
+                "chmod -R 755 /usr/local/sbin",
```

Comment:
Why do we apply the second time at this line and line 60?

Reply:
Just to return permissions back to default one from 777

Reply:
Why do we need to change permissions with sudo?